requirements.txt

```diff
@@ -1 +1 @@
-certora-cli==3.6.4
+certora-cli==4.3.1
```

Comment:
If we merge this will all the existing specs stop working?

Reply:
They already don't work ... because 3.6.4 is no longer supported

Reply:
We'll have to do a "bump FV to new format" task during the quality control sprint. I can take care of it.